b3sum/tests/test.rs

```diff
@@ -93,3 +105,11 @@ fn test_length_without_value_is_an_error() {
         .run();
     assert!(result.is_err());
 }
+
+#[test]
+fn test_raw_with_multi_files_is_an_error() {
+    let result = cmd!(b3sum_exe(), "--raw", "file1", "file2")
```

Comment:
I think this test could fail to test what it looks like, because the fact that `file1` and `file2` don't exist will lead to another type of error. Could you create a couple temporary files (with `NamedTempFile` as in other tests) to prevent this error from happening? You should be able to confirm that if you comment out the `len() > 1` check in main, that this test no longer gets and error and starts failing.

Reply:
Good catch!  

This is now fixed.